mdx-layout-css/src/index.tsx

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import { MDXProvider } from '@mdx-js/react';
-import { Playground } from '~/playground/dist/Playground.js';
-import { Showcases } from '~/showcases/dist/Showcases.js';
-import { Props } from '~/props/dist/Props.js';
-import { Layout as LayoutComponent } from './Layout';
-import { StylesheetSwitch } from './StylesheetSwitch';
+import { Playground } from '~/playground';
+import { Showcases } from '~/showcases';
+import { Props } from '~/props';
+import { CoreLayout, StylesheetSwitch } from '~/mdx-layout-core';
 
-export const Layout = (props) => (
+export const CssLayout = (props) => (
```

Comment:
Do we need to change the classname here? since while importing you will have `import { Layout } from '~/mdx-layout-css'`  I think it's enough no ?

Reply:
That was the my first version but @georges-gomes suggested specific naming
"GeorgesYesterday at 5:40 PM
call them "LayoutCore" and "LayoutCss" to differentiate them
otherwise it's layout everywhere"

Reply:
I don't have strong opinion on this, but I think there will be only one Layout. so I saw this more or less like the same as ThemeProvider, it's theme provider everywhere too. But I don't mind